modules/rostests/apitests/gdi32/AddFontResource.c

```diff
@@ -11,7 +11,7 @@
 
 void Test_AddFontResourceA()
 {
-    CHAR szFileNameA[MAX_PATH];
+    CHAR szFileNameA[MAX_PATH*2 + 3];
```

Comment:
I'm still uneasy with such changes...
A path longer than MAX_PATH wouldn't properly be handled by ReactOS/Windows.
Furthermore, you didn't the memory zeroing performed after.
And the operations performed on that big buffer are unclear.
It would be nice to have @katahiromz's opinion on this.

Reply:
According to MSDN, `AddFontResourceA` function supports multiple filenames:
> To add a font whose information comes from several resource files, have lpszFileName point to a string with the file names separated by a "|" --for example, abcxxxxx.pfm | abcxxxxx.pfb.

And that's what we test here:
```
sprintf(szFileNameA,"%s|%s",szFileNameFont1A, szFileNameFont2A);
```

Reply:
Fixed RtlZeroMemory call btw

Reply:
OK, perfect. Thanks for the feedback :-)